apps/demo-app/src/app/app.component.html

```diff
@@ -10,7 +10,7 @@
 
     <a mat-flat-button color="primary" target="_blank" class="header-element"
       href="https://github.com/enigmatry/entry-angular-building-blocks">
-      <img class="github-icon" src="../assets/images/github-circle-white-transparent.svg" />
+      <img class="github-icon" src="../assets/images/github-circle-white-transparent.svg" alt="GitHub white icon" />
```

Comment:
The purpose of this icon is to link to the GitHub source repository for Entry packages. Maybe to consider changing the alt text provides more context about the purpose of the image, which is helpful for users who rely on screen readers such as "View Enigmatry source code repository."